tests/cpydiff/modules_json_nonserializable.py

```diff
@@ -8,8 +8,7 @@
 
 a = bytes(x for x in range(256))
 try:
-    z = json.dumps(a)
-    x = json.loads(z)
-    print("Should not get here")
+    z = json.dumps(json)
+    print("Produced non-conforming json string", repr(z))
```

Comment:
I think the original test was testing that `bytes` are not serialisable, which would be more of a subtle difference between CPython and MicroPython than trying to serialise a module.

Suggest to simply do:
```py
try:
    print(json.dumps(b"shouldn't be able to serialise this")
except TypeError:
    print("TypeError")
```